dev/com.ibm.websphere.appserver.jsfApiStub-2.2/bnd.bnd

```diff
@@ -0,0 +1,22 @@
+ -include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Description: WAS JSF 2.2 API Stub, version ${bVersion}
+Bundle-Name: WAS JSF 2.2 API Stub
+Bundle-SymbolicName: com.ibm.ws.jsf.2.2
+Bundle-Version: ${bVersion}.${libertyBundleMicroVersion}.apistub
+Eclipse-ExtensibleAPI: true
+
+publish.wlp.jar.suffix: dev/api/third-party
+
+-exportcontents: \
+  com.ibm.ws.jsf.cdi, \
+  com.ibm.ws.jsf.config, \
+  com.ibm.ws.jsf.config.annotation, \
+  com.ibm.ws.jsf.config.resource, \
+  com.ibm.ws.jsf.ext, \
+  com.ibm.ws.jsf.extprocessor, \
+  com.ibm.ws.jsf.spi, \
+  com.ibm.ws.jsf.spi.impl
+
+-fixupmessages.missingexport: "Used bundle version * for exported package";is:=ignore 
```

Comment:
Is this required here and in the thirdparty bnd?

Reply:
It is - when packages lack explicit exports, the bnd tooling throws an error like `error  : Used bundle version 1.0.## for exported package com.ibm.X~`

In this stub case, since `-exportcontents` is used the actual package versions aren't picked up, and we want to use the generated package versions.  In the thirdparty case, the `org.apache.myfaces.*` packages we export don't have explicit versions, and again we want and intend to use the generated versions.